schema/record-package-schema.json

```diff
@@ -122,7 +122,7 @@
   "definitions": {
     "Record": {
       "title": "Record",
-      "description": "An OCDS record must provide a list of all the existing OCDS releases related to a single contracting process and should provide a compiled release containing the current state of all fields in the release schema. An OCDS record may also provide a versioned history of all changes to the data in the compiled release.",
+      "description": "A record aggregates the releases related to a contracting process. A record must include a `releases` list, should include a `compiledRelease` and may include a `versionedRelease`. There should be a single record per contracting process per [distribution](https://www.w3.org/TR/vocab-dcat-2/#Class:Distribution), where a distribution might be a specific API endpoint or a specific bulk download file.",
```

Comment:
The "must" statement is enforced by `required`, which I don't think we typically repeat in field descriptions. I wonder whether the should statement can be moved to the `compiledRelease` description. The remaining "may" statement is not necessary.

Reply:
I guess the original intention was to have all the statements in one place. However, that need is met by `records_reference.md` so I've made the changes you suggest in 5521210ad1d01fefdd482e40064cce79cd7d4784